### ISSUE TITLE
warn user when uses prerendered pages with revalidate: 0

### ISSUE THREAD

**Issue by eneajaho** [eneajaho/ngx-isr#22]

**Comment by jdgamble555** [comment]
The logging doesn't seem to be working at all with `revalidate: 0` either!

J

**Comment by eneajaho** [comment]
Hello @jdgamble555 

Can you explain more what do you mean by that?

**Comment by jdgamble555** [comment]
Yes.  When I set `revalidate: 5` I can see the logging in my server console.  However, there is no logging when `revalidate: 0`.

Thanks,

J

**Comment by eneajaho** [comment]
Hello @jdgamble555 
I just tested this and here's what I get. 

<img width="1512" alt="Screenshot 2022-09-27 at 5 45 49 PM" src="https://user-images.githubusercontent.com/25394362/192573237-2fef0694-7a88-4ec2-b63d-df143f169f0d.png">

Can you provide more info please? 
Are you using the default cache handler or the FileSystemCacheHandler? 

Thanks.

**Comment by eneajaho** [comment]
Hello @jdgamble555 , sorry for late reply, I've been busy with other work. 

I'm using the default cache too on that example. Did you manage to solve your issue? 
If not, can you provide me a reproduction repo that I can clone and check it? 

Thanks.

**Comment by jdgamble555** [comment]
I think I got it working. I migrated my old app to a new app and have no problems. I'm not sure what happened. Thanks for your time and work on this.

Would be cool to copy this feature in SvelteKit btw. I imagine if it works in Vite, it could work anywhere.

Thanks,

J
